### ISSUE TITLE
Docker Compose The path must be absolute

### ISSUE THREAD

**Issue by groogiam** [dotnet/aspnetcore#39286]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the bug

When running via docker compose via Visual Studio container tools in Visual Studio 17.0.4 the ASP.NET core site will not start with

```
System.ArgumentException: ‘The path must be absolute.’
at Microsoft.Extensions.FileProviders.PhysicalFileProvider…ctor(String root, ExclusionFilters filters)
at Microsoft.Extensions.FileProviders.PhysicalFileProvider…ctor(String root)
at Microsoft.AspNetCore.Hosting.StaticWebAssets.StaticWebAssetsLoader.<>c.<UseStaticWebAssetsCore>b__1_0(String contentRoot)
at Microsoft.AspNetCore.StaticWebAssets.ManifestStaticWebAssetFileProvider…ctor(StaticWebAssetManifest manifest, Func`2 fileProviderFactory)
at Microsoft.AspNetCore.Hosting.StaticWebAssets.StaticWebAssetsLoader.UseStaticWebAssetsCore(IWebHostEnvironment environment, Stream manifest)
at Microsoft.AspNetCore.Hosting.StaticWebAssets.StaticWebAssetsLoader.UseStaticWebAssets(IWebHostEnvironment environment, IConfiguration configuration)
at Microsoft.AspNetCore.WebHost.<>c.<ConfigureWebDefaults>b__9_0(WebHostBuilderContext ctx, IConfigurationBuilder cb)
at Microsoft.AspNetCore.Hosting.WebHostBuilder.BuildCommonServices(AggregateException& hostingStartupErrors)
at Microsoft.AspNetCore.Hosting.WebHostBuilder.Build()
at AppPotion.Web.Program.Main(String[] args) in C:\Users\Greg\source\repos\AppPotionV2\src\AppPotion.Web\Program.cs:line 12
```

These two issues appear to be the same. Both are closed but the issue does not look like it was ever fixed.
https://github.com/dotnet/aspnetcore/issues/35285
https://github.com/dotnet/aspnetcore/issues/37876

### Expected Behavior

Docker Compose should work as it did in .NET 5

### Steps To Reproduce

I'm working on a clean repro.

### .NET Version

6.0.101

**Comment by TanayParikh** [comment]
Based on https://github.com/dotnet/aspnetcore/issues/37876, it appears the workaround is disabled docker-compose V2 and then rebuilding the solution. 

The root cause is mentioned [here](https://github.com/dotnet/aspnetcore/issues/37876#issuecomment-962213965) and [here](https://developercommunity.visualstudio.com/t/Not-able-to-debug-a-Blazor-Server-App-wi/1556545). Given the issue is due to a regression in docker-compose, there isn't much we can do until docker releases a fix.

**Comment by TanayParikh** [comment]
Ah, looks like this'll be fixed in VS 17.1 Preview 3: https://github.com/microsoft/DockerTools/issues/320

**Comment by groogiam** [comment]
@TanayParikh Thanks for the quick response.
